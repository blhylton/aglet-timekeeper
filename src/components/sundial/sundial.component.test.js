import React from 'react';
import renderer from 'react-test-renderer';
import Sundial from './sundial.component';

it('renders correctly', () => {
  const tree = renderer.create(
    <Sundial
      phaseOfMoon={1}
      sky={'day'}
      rotation={360}
    />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
