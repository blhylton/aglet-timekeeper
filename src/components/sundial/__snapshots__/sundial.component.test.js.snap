// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sundial"
>
  <div
    className="sky day"
  >
    <div
      className="stars"
    />
    <div
      className="sunMoon"
      style={
        Object {
          "transform": "rotate(360deg)",
        }
      }
    >
      <div
        className="sun"
        style={
          Object {
            "transform": "rotate(-360deg)",
          }
        }
      />
      <div
        className="moon moon1"
        style={
          Object {
            "transform": "rotate(-360deg)",
          }
        }
      />
    </div>
  </div>
</div>
`;
